torchci/rockset/commons/__sql/flaky_tests.sql

```diff
@@ -0,0 +1,27 @@
+SELECT
+	test_run.name,
+	test_run.classname as suite,
+	test_run.file,
+	SUM(ELEMENT_AT(JSON_PARSE(REPLACE(test_run.skipped.message, 'True', 'true')), 'num_green')) as numGreen,
+	SUM(ELEMENT_AT(JSON_PARSE(REPLACE(test_run.skipped.message, 'True', 'true')), 'num_red')) as numRed,
+	ARRAY_AGG(job.name) as jobNames,
+	ARRAY_AGG(job.id) as jobIds,
+	ARRAY_AGG(workflow.id) as workflowIds,
+	ARRAY_AGG(workflow.name) as workflowNames,
+	ARRAY_AGG(workflow.head_branch) as branches,
+    ARRAY_AGG(test_run.workflow_run_attempt) as runAttempts
+FROM
+    commons.workflow_job job 
+    INNER JOIN commons.test_run ON test_run.job_id = job.id HINT(join_strategy=lookup)
+    INNER JOIN commons.workflow_run workflow ON job.run_id = workflow.id 
+WHERE 
+	NOT test_run.skipped IS NULL
+    AND STRPOS(test_run.skipped.message, 'num_red') > 0  
+    AND test_run._event_time > (CURRENT_TIMESTAMP() - HOURs(:numHours)) 
+    AND test_run.name LIKE :name
```

Comment:
Faster to use `=` unless you are actually using the features of `LIKE` (like wildcards)